### DISCUSSION TITLE
Developer Intern (May - 4 months) - IBM Consulting

### DISCUSSION THREAD

**Discussion by Hamzenium** [jenndryden/Canadian-Tech-Internships-and-New-Grad-2025#186] [Q&A]
Hey Guys,

I hope everyone is doing well.
I was wondering if anyone has received an assessment for this role, I received an email that said I would be receiving an email for the assessment but it has been more than 10 days and I have not received any update.  

Would appreciate it, if any could suggest what I should do.

**Comment by kanjurer** [comment]
Same, but I think I have given up on IBM personally. Even with 100% OA scores they never follow up, maybe it is just too competitive

**Reply by Hamzenium** [reply]
How long after receiving the email with the accommodation form did you receive the assessment, and how was it? To be honest, I have given up because they never follow up on the application; I even tried contacting recruiters, but they did not respond.

**Reply by ypatel2022** [reply]
I dont think the IBM OA means anything. I didnt even do good on mine (I didnt get all test cases) and still got an interview + offer.

**Reply by kanjurer** [reply]
Bro 😭 
Congratulations but damn they shouldn't send OA if they don't consider the scores

**Reply by Hamzenium** [reply]
Was this for the summer 2024 position, the offer you received ?

**Reply by ypatel2022** [reply]
It was another soft dev intern position for summer 2024
